RELEASES.md

```diff
@@ -383,7 +399,6 @@ The deprecated features are shown in the following table:
 |----------------------------------------------------------------------------------|---------------------|----------------------------|------------------------------------------|
 | Runtime V1 API and implementation (`io.containerd.runtime.v1.linux`)             | containerd v1.4     | containerd v2.0 ✅         | Use `io.containerd.runc.v2`              |
 | Runc V1 implementation of Runtime V2 (`io.containerd.runc.v1`)                   | containerd v1.4     | containerd v2.0 ✅         | Use `io.containerd.runc.v2`              |
-| config.toml `version = 1`                                                        | containerd v1.5     | containerd v2.0 ✅         | Use config.toml `version = 2`            |
```

Comment:
The table should retain completed items

Reply:
(v1.x users still refer to this document in the main branch)

Reply:
Leaving it in the deprecation table is confusing. I mentioned that it is no longer deprecated in the config section. Users shouldn't have to think about it much anymore now that a utility is available to migrate if they want to, but don't need to.

Reply:
> (v1.x users still refer to this document in the main branch)

Version 1 still works in v1.x though, what does a user need to see here?